_modules/forests.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
The first function tests the existence of a forest.

Now, the next one will list all the nodes in a forest.

And to get the list of the forest we've already pillar['forests'] available.

So all the functions will probably be related to an unique forest.

As such, singular is better.